File: w06/d03/Classwork/pos_neg.rb
# Given 2 integers, 
# return true if one is negative and one is positive.

# posNeg(1, -1)
# // true
# posNeg(-1, 1)
# // true
# posNeg(-4, -5)
# // false

def posneg(x,y) 
	if ( x < 0 && y > 0 ) || ( x > 0 && y < 0 )
		return true
	else
		return false
	end
end

posneg(1, -1)
posneg(-1, 1)
posneg(-4, -5)


def posneg(x,y) 
	if ( x < 0 && y > 0 ) || ( x > 0 && y < 0 )
		return true
	else
		return false
	end
end

posneg(1, -1)
posneg(-1, 1)
posneg(-4, -5)

File: w06/d03/Classwork/posneg.js
function pos_neg(x, y, z) {
	if ((x * y < 0 && z == false) || (x < 0 && y < 0 && z == true))
	{
		console.log(true)
	} else {
		console.log(false)
	};
};

pos_neg (-1, 2, false)
pos_neg (1, 3, false)
pos_neg (-1, -4, true)